packages/svelte-ux/src/lib/utils/date.ts

```diff
@@ -57,15 +57,27 @@ export function getPeriodTypeName(periodType: PeriodType) {
   return getPeriodTypeNameWithLocale(defaultLocale, periodType);
 }
 
+// util to make sure we have handled all enum cases
+function assertNever(x: never): never {
+  throw new Error(`Unhandled enum case: ${x}`);
+}
+
```

Comment:
Should we move this to `typeHelpers`?  I know the error message isn't exactly generic though...

Reply:
It's not really about the error message, it's more that when we will be able to have `pnpm check`, type will complain at the `switch` level and showing what case is not handled.

I move it to `typeHelpers` and added a bit of doc on top